addons/mail/static/src/core/common/persona_model.js

```diff
@@ -61,6 +61,13 @@ export class Persona extends Record {
     notification_preference;
     isAdmin = false;
     isInternalUser = false;
+    storeAsPresence = Record.one("Store", {
+        compute() {
+            if (this.channelMembers.some(({ thread }) => thread.model === "discuss.channel")) {
+                return this._store;
+            }
+        },
+    });
```

Comment:
`channelMembers` are necessarily linked to a `discuss.channel` conversation, so the `.some()` looks useless. Only `length !== 0` is enough.